js/classes/inventory.js

```diff
@@ -32,7 +36,8 @@ class Inventory {
      */
     getAmmo(type) {
         for (let i = 0; i < this.ammoList.length; i++) {
-            if (this.ammoList[i].type == type) {
+            let ammo = this.ammoList[i];
+            if (ammo.type == type && this) {
```

Comment:
I'm not sure I understand the reason for this change. What's the point of checking for `this`?